src/deployment.ts

```diff
@@ -87,6 +87,46 @@ export class Deployment {
         return Promise.resolve();
     }
 
+    @vsCommand('openshift.deployment.scale')
+    static async scalingPod(component: KubernetesObject): Promise<void> {
+        if (!component) {
+            return;
+        }
+        const pods: DeploymentPodObject[] = await OpenShiftExplorer.getInstance().getPods(component);
+        const runnngPodsLength = pods.filter((pod) => pod.status.phase === 'Running').length;
+        let count = runnngPodsLength.toString();
+        count = await inputValue(`How many replicas would you like to scale ${component.kind}/${component.metadata.name}?`,
+            count,
+            false,
+            (value: string) => {
+                const trimmedValue = value.trim();
+                const number = Number(trimmedValue);
+                if (trimmedValue.length === 0) {
+                    return 'Scale value cannot be empty';
+                }
+                if (isNaN(number)) {
+                    return 'Scale value should be a number';
+                } else  if (parseInt(trimmedValue, 10) < 0) {
+                    return 'Scale value should be equal or greater than zero';
+                } else if (Deployment.checkFloat(trimmedValue)) {
```

Comment:
I am allowed to scale to `2.0`, but when I do this, it fails, since the string `2.0` is passed to the `oc` command that scales the deployment.